definitions/infra-awsapigatewayapi/golden_metrics.yml

```diff
@@ -28,6 +30,7 @@ errors4XxAnd5Xx:
       eventName: entityName
 latencyMs:
   title: Latency (ms)
+  unit: SECONDS
```

Comment:
from the title seems this is on miliseconds right? so we should convert it into seconds